caffe2/python/pybind_state_ideep.cc

```diff
@@ -105,53 +103,80 @@ class IDeepFeeder : public BlobFeederBase {
       return itensor::data_type::data_undef;
   }
 
- public:
-   void FeedTensor(
-       const DeviceOption& option,
-       PyArrayObject *original_array,
-       itensor *tensor) {
-     PyArrayObject *array = PyArray_GETCONTIGUOUS(original_array);
-     auto g = MakeGuard([&]() {Py_XDECREF(array); });
-
-     const auto npy_type = PyArray_TYPE(array);
-     const TypeMeta& meta = NumpyTypeToCaffe(npy_type);
-     CAFFE_ENFORCE(
+public:
+  void FeedTensor(
+      const DeviceOption &option,
+      PyArrayObject *original_array,
+      itensor *tensor) {
+    PyArrayObject *array = PyArray_GETCONTIGUOUS(original_array);
+    auto g = MakeGuard([&]() { Py_XDECREF(array); });
+    const auto npy_type = PyArray_TYPE(array);
+    const TypeMeta &meta = NumpyTypeToCaffe(npy_type);
+    CAFFE_ENFORCE(
```

Comment:
Consider to use CAFFE_ENFORCE_NE which will provide more debug info when failed.